pkg/utils/algo/algos.go

```diff
@@ -0,0 +1,13 @@
+package algo
+
+import "golang.org/x/exp/constraints"
+
+// Include checks if a string existing in a string slice and returns true or false.
+func Include[T constraints.Ordered](target T, values []T) bool {
```

Comment:
Consider using github.com/samber/lo for functional interfaces like this.

Reply:
Changed to the powerful lib.